addons/mail/static/src/widgets/discuss/discuss.js

```diff
@@ -63,6 +63,7 @@ const DiscussWidget = AbstractAction.extend({
     async willStart() {
         await this._super(...arguments);
         this.env = Component.env;
+        await this.env.messagingCreatedPromise;
```

Comment:
The discuss part of the fix is currently not covered by a test.

Reply:
Indeed, but this is annoying to make clean test with how works `mail/test_utils.js:start()`.

Idea would consist of testing that discuss is not mounted when messaging not created, and then becomes mounted when created. It's not desirable to make `start()` more heavy than it already is, so that it supports resolution of function with discuss not yet mounted.

Would prefer cleaning `start()` before doing that (i.e. decoupling setup of `env` and setup of components/widgets). I much prefer not doing test than making a bad test (and artificially make `start()` more complicated than it should be).

Reply:
ok to ignore for now if too hard

Reply:
task-2326470